sdk/python/feast/feature_store.py

```diff
@@ -153,26 +160,131 @@ def get_historical_features(
         )
         return job
 
+    def get_online_features(
+        self, feature_refs: List[str], entity_rows: List[Dict[str, Any]],
+    ) -> OnlineResponse:
+        """
+        Retrieves the latest online feature data.
+        Args:
+            feature_refs: List of feature references that will be returned for each entity.
+                Each feature reference should have the following format:
+                "feature_table:feature" where "feature_table" & "feature" refer to
+                the feature and feature table names respectively.
+                Only the feature name is required.
+            entity_rows: A list of dictionaries where each key-value is an entity-name, entity-value pair.
+        Returns:
+            OnlineResponse containing the feature data in records.
+        Examples:
+            >>> from feast import FeatureStore
+            >>>
+            >>> store = FeatureStore(repo_path="...")
+            >>> feature_refs = ["sales:daily_transactions"]
+            >>> entity_rows = [{"customer_id": 0},{"customer_id": 1}]
+            >>>
+            >>> online_response = store.get_online_features(
+            >>>     feature_refs, entity_rows, project="my_project")
+            >>> online_response_dict = online_response.to_dict()
+            >>> print(online_response_dict)
+            {'sales:daily_transactions': [1.1,1.2], 'sales:customer_id': [0,1]}
+        """
 
-def _get_requested_feature_views(
+        response = self._get_online_features(
+            feature_refs=feature_refs,
+            entity_rows=_infer_online_entity_rows(entity_rows),
+            project=self.config.project,
+        )
+
+        return OnlineResponse(response)
+
+    def _get_online_features(
```

Comment:
Possible to make this a function so that it's easier to test?

`_get_online_features(provider, all_feature_views, entity_rows, feature_refs, project)`?

Reply:
Hmm I'm generally a fan of standalone functions wherever possible too, but here I'd need to pass it basically every property of `FeatureStore` (`provider`, `project` and `registry`). So I figured it is okay to keep it as a method? Don't feel strongly though.

Reply:
Youd not need to pass project/registry right? You only need to provide the list of feature views. 

I just want to keep the FeatureStore class as simple as possible. It's starting to bloat already. If we're going to add an OnlineStore class as well, then we'll have 3x `get_online_features()` methods. Seems like having this function in the online_store.py would make more sense (I guess the same applies to my historical retriever).

Reply:
Ah i see you don't want another giant Client, makes sense. How about I do another PR to split out OnlineStore and I can move this stuff there then?

Reply:
Cool, thanks!